phoebe/atmospheres/passbands.py

```diff
@@ -38,7 +36,7 @@
 
 # define the URL to query for online passbands.  See tables.phoebe-project.org
 # repo for the source-code of the server
-_url_tables_server = 'http://tables.phoebe-project.org'
+_url_tables_server = 'https://staging.phoebe-project.org'
```

Comment:
this definitely shouldn't be merged, right?

Reply:
It should be merged into feature-blending. Staging is running the blending version of phoebe within the server.